assets/packages/adminbasics/src/parts/save.js

```diff
@@ -175,8 +175,8 @@ const SaveController = () => {
                     if ($form.data('isvuecomponent') == true) {
                         LS.EventBus.$emit('componentFormSubmit', button)
                     } else {
-                        $form.find('[type="submit"]').first().trigger('click');
                         displayLoadingState(this);
```

Comment:
displayLoadingState must be in $form submit event. Never on user action.

Reply:
Well, but that would impact many screens... seems too much of a refactoring for this bug.

Reply:
Oh ! It's for all save admin form ????

Then it's a biggest issue, it show here : but it muts be really when form is submitted to show spinner. Never user actin.

Reply:
@c-schmitz ?